content/docs/reference/pkg/aws/inspector/getrulespackages.md

```diff
@@ -0,0 +1,260 @@
+                
+                <code>List&lt;string&gt;</code>
+            </td>
+            <td class="align-top">{{% md %}} A list of the AWS Inspector Rules Packages arns available in the AWS region.
+ {{% /md %}}
+
+            
+            </td>
+        </tr>
+    
+        <tr>
+            <td class="align-top">Id</td>
+            <td class="align-top">
+                
+                <code>string</code>
+            </td>
+            <td class="align-top">{{% md %}} id is the provider-assigned unique ID for this managed resource.
+ {{% /md %}}
+
+            
+            </td>
+        </tr>
+    
+    </tbody>
+</table>
+
+
+{{% /lang %}}
+
+
+{{% lang go %}}
+
+
+<table class="ml-6">
+    <thead>
+        <tr>
+            <th>Argument</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+    
+        <tr>
+            <td class="align-top">Arns</td>
+            <td class="align-top">
+                
+                <code>[]string</code>
+            </td>
+            <td class="align-top">{{% md %}} A list of the AWS Inspector Rules Packages arns available in the AWS region.
+ {{% /md %}}
+
+            
+            </td>
+        </tr>
+    
+        <tr>
+            <td class="align-top">Id</td>
+            <td class="align-top">
+                
+                <code>string</code>
+            </td>
+            <td class="align-top">{{% md %}} id is the provider-assigned unique ID for this managed resource.
+ {{% /md %}}
+
+            
+            </td>
+        </tr>
+    
+    </tbody>
+</table>
+
+
+{{% /lang %}}
+
+
+{{% lang nodejs %}}
+
+
+<table class="ml-6">
+    <thead>
+        <tr>
+            <th>Argument</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+    
+        <tr>
+            <td class="align-top">arns</td>
+            <td class="align-top">
+                
+                <code>string[]</code>
+            </td>
+            <td class="align-top">{{% md %}} A list of the AWS Inspector Rules Packages arns available in the AWS region.
+ {{% /md %}}
+
+            
+            </td>
+        </tr>
+    
+        <tr>
+            <td class="align-top">id</td>
+            <td class="align-top">
+                
+                <code>string</code>
+            </td>
+            <td class="align-top">{{% md %}} id is the provider-assigned unique ID for this managed resource.
+ {{% /md %}}
+
+            
+            </td>
+        </tr>
+    
+    </tbody>
+</table>
+
+
+{{% /lang %}}
+
+
+{{% lang python %}}
+
+
+<table class="ml-6">
+    <thead>
+        <tr>
+            <th>Argument</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+    
+        <tr>
+            <td class="align-top">arns</td>
+            <td class="align-top">
+                
+                <code>list[string]</code>
```

Comment:
Also, when specifying a typehint for a list in Python, the `List` is generally upper-case, e.g. `List[str]`.

Reply:
I fixed this in the latest commit in https://github.com/pulumi/pulumi/pull/4085.